wiki/en/Getting-Started.md

```diff
@@ -43,7 +43,7 @@ When you open Jamulus, you will see a window which looks like this:
 
 ## Setting up a profile
 
-First, let others know who you are. Click on “View” (bar on the top) > “My Profile…” Now you will see this:
+First, let others know who you are. Click on the “Settings” button on the bottom left and navigate to “My Profile…”. Now you will see this:
```

Comment:
I've been trying to use the less specialised "go to" rather than "navigate to" for this.

Reply:
Yeah. Just edit it as you‘d prefer.

Reply:
```suggestion
First, let others know who you are. Click on the “Settings” button on the bottom left and go to “My Profile…”. Now you will see this:
```